src/Surfnet/Stepup/IdentifyingData/Entity/IdentifyingData.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\Stepup\IdentifyingData\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="Surfnet\Stepup\IdentifyingData\Entity\IdentifyingDataRepository")
+ * @ORM\Table(name="identity_identifying_data")
+ */
+class IdentifyingData
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(length=36)
+     *
+     * @var string
+     */
+    public $uuid;
+
+    /**
+     * @ORM\Column(length=255)
+     *
+     * @var string
+     */
+    public $commonName;
```

Comment:
I have a dark brown suspicion this should actually be a `CommonName` (with corresponding Doctrine Type). Same goes for the e-mail field below.